src/main/views/steps-to-making-your-claim.njk

```diff
@@ -7,19 +7,103 @@
 {% endblock %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">{{ h1 }}</h1>
-  <h2 class="govuk-heading-l">{{ section1.title }}</h2>
-  <hr class="govuk-section-break govuk-section-break--visible">
-  <a href="{{ PageUrls.DOB_DETAILS }}" class="govuk-link">{{ section1.link1Txt }}</a>
-  <hr class="govuk-section-break govuk-section-break--visible">
-  <a href="{{ PageUrls.ADDRESS_DETAILS }}" class="govuk-link">{{ section1.link2Txt }}</a>
-  <hr class="govuk-section-break govuk-section-break--visible">
-  <a href="{{ PageUrls.UPDATE_PREFERENCES }}" class="govuk-link">{{ section1.link3Txt }}</a>
-  <hr class="govuk-section-break govuk-section-break--visible">
+
+  <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">{{h1}} TODO</h1>
+  <table class="govuk-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">{{ section1.title }}</th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__cell">  
+        <a href="{{ PageUrls.DOB_DETAILS }}" class="govuk-link">
+        {{ section1.link1Txt }}
+        </a>
+        </th>
+        <th scope="row" class="govuk-table__cell">TODO</th>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__cell">
+        <a href="{{ PageUrls.ADDRESS_DETAILS }}" class="govuk-link">
+        {{ section1.link2Txt }}
+        </a>
+        </th>
+        <th scope="row" class="govuk-table__cell">TODO</th>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__cell">
+        <a href="{{ PageUrls.UPDATE_PREFERENCES }}" class="govuk-link">
+        {{ section1.link3Txt }}
+        </a>
+        </th>
+        <th scope="row" class="govuk-table__cell">TODO</th>
+      </tr>
+    </tbody>
+  </table>
+  
+  <table class="govuk-table">
```

Comment:
I'm seeing a lot of html repeated on this page. Could you please investigate using loops so we can cut down the html.
The nunjucks documentation has some guides on how this can be done - https://mozilla.github.io/nunjucks/templating.html#for

Reply:
taken into advisement and reduced code reuse